CHANGES.md

```diff
@@ -2,11 +2,13 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.351 Jan 30 2024
+- Deprecate notify resource
+
 ## 0.0.350 Jan 29 2024
 - [OCM-5802] Add `ExternalAuthConfig` to `Cluster` model.
 
-## 0.0.349 Jan 28 2024
--  [OCM-5561] Remove redundant fields from /notify_details
+## 0.0.349 Jan 28 2024Notify from /notify_details
```

Comment:
@Lir10 why is this change required?

Reply:
good catch .. no idea where that came from.
updated